pkgs/top-level/perl-packages.nix

```diff
@@ -13885,6 +13885,24 @@ let
      };
   };
 
+  NetServerSSPrefork = buildPerlPackage {
+     pname = "Net-Server-SS-PreFork";
+     version = "0.06pre";
```

Comment:
```suggestion
     version = "unstable-2017-06-08";
```

Reply:
@volth Sorry, I missed your previous review+suggestion regarding version.  I assume that `0.06pre` is preferred over `unstable-2017-06-08`?